src/browser/index.tsx

```diff
@@ -24,4 +24,19 @@ import AppInit, { setupSentry } from './AppInit'
 import './init'
 
 setupSentry()
-ReactDOM.render(<AppInit />, document.getElementById('mount'))
+;(async () => {
+  const doesPreferQuery = localStorage.getItem('prefersOldBrowser') === 'false'
+
+  const response = await fetch('/preview/manifest.json')
```

Comment:
I think `fetch` can throw, so I think it'd be good with  try { } catch { } here

Reply:
This also needs to be `fetch('./preview/manifest.json')` to handle the bundled case where it's under `/browser/preview`. And also to handle if users have set up their own custom hosting!

Reply:
Perfect point, changed it accordingly!